### PR TITLE
Updating pkg to latest for 0.10 release.

### PR DESCRIPTION
**Pull request by n3wscott**
## Proposed Changes

- Prepping for 0.10 release. Update knative.dev/pkg to latest from branch release-0.10.
- Update to yaml based webhook registration.

**Release Note**

```release-note
NONE
```

/cc @mattmoor

### CODE REVIEW

File: couchdb/source/config/500-webhook-configuration.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    contrib.eventing.knative.dev/release: devel
+webhooks:
+- admissionReviewVersions:
+  - v1beta1
+  clientConfig:
+    service:
+      name: couchdb-webhook
+      namespace: knative-eventing
+  failurePolicy: Fail
+  name: webhook.couchdb.eventing.knative.dev
```
Comment (mattmoor-sockpuppet) [inline couchdb/source/config/500-webhook-configuration.yaml]: Add trailing newline:
```suggestion
  name: webhook.couchdb.eventing.knative.dev

```

File: kafka/channel/config/500-webhook-configuration.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    contrib.eventing.knative.dev/release: devel
+webhooks:
+- admissionReviewVersions:
+  - v1beta1
+  clientConfig:
+    service:
+      name: kafka-webhook
+      namespace: knative-eventing
+  failurePolicy: Fail
+  name: webhook.kafka.eventing.knative.dev
```
Comment (mattmoor-sockpuppet) [inline kafka/channel/config/500-webhook-configuration.yaml]: Add trailing newline:
```suggestion
  name: webhook.kafka.eventing.knative.dev

```

File: couchdb/source/config/500-webhook-configuration.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: webhook.couchdb.messaging.knative.dev
```
Comment (mattmoor) [inline couchdb/source/config/500-webhook-configuration.yaml]: this doesn't match above

Comment (n3wscott) [reply couchdb/source/config/500-webhook-configuration.yaml]: fixed.

**Pull request merged**